web/README.md

```diff
@@ -0,0 +1,440 @@
+see this [README](https:githubcomcloudspannerecosystemharbourbridge#harbourbridge-turnkey-spanner-evaluation). Note that this feature is experimental, and should not be used for
+production database.
+
+### Starting web server for HarbourBridge
+
+The following example assume `harbourbridge` has been added to your PATH
+environment variable.
+
+HarbourBridge's Web API feature can be used with all the driver modes available,
+using mysql or postgres dump or direct connection.
+
+To start HarbourBridge web server, run:
+
+```sh
+harbourbridge --web
+```
+
+The tool will be available on port 8080
+
+You can go to `<base-url>/` to visit the UI of the tool.
+
+    eg: localhost:8080/
+
+For more details on how to use the UI, you can visit: `<base-url>/userManual.html`.
+
+    eg: localhost:8080/userManual.html
+
+<ins>**Note:**</ins>
+
+The `pg_dump` and `mysqldump` drivers cannot be used for data migration if the
+Spanner schema has interleaved tables. Note that the `postgres` or `mysql` drivers
+do not have this restriction -- consider using these as an alternative.
+
+## APIs
+
+These are the REST APIs and their details:
+
+### Connect
+
+`/connect` is a POST API used to configure direct connection to a database.
+The request body contains connection details and the driver name.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Connection information along with driver name.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "Host": "localhost",
+  "Port": "5432",
+  "Database": "dbname",
+  "User": "user",
+  "Password": "password"
+}
+```
+
+#### Response body
+
+No response body is returned.
+
+### Convert
+
+(1) `/convert/infoschema` is a GET API followed by `/connect` API to convert using
+infoschema mode. It returns the schema conversion state in json format.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Conv struct in JSON format.
+
+(2) `/convert/dump` is a POST API used to perform schema conversion on a dump file.
+The request body contains the path to the dump file.
+
+#### Method
+
+`POST`
+
+#### Request body
+
+Provide driver and path to dump file.
+
+Example
+
+```json
+{
+  "Driver": "postgres",
+  "Path": "/path/to/dumpFile"
+}
+```
+
+#### Response body
+
+Conv struct in JSON format.
+
+### DDL
+
+`/ddl` is a GET API which must be used after using conversion APIs (i.e, `/connect`
+or `/convert`). This API returns the DDL statements of the converted schema.
+
+#### Method
+
+`GET`
+
+#### Request body
+
+No request body is needed.
+
+#### Response body
+
+Table wise DDL statements.
+
+Example
+
+```json
+{
+  "Albums": "--\n-- Spanner schema for source table Albums\n--\nCREATE TABLE Albums (\n    SingerId INT64 NOT NULL, -- From: SingerId smallint(6)\n    AlbumId INT64 NOT NULL,  -- From: AlbumId smallint(6)\n    AlbumTitle STRING(50)    -- From: AlbumTitle varchar(50)\n) PRIMARY KEY (SingerId, AlbumId)"
```

Comment:
Is this formatting here what was intended?
It gets rendered on a single line, and isn't what we show for json examples elsewhere?

Reply:
this is a result of an API, so i thought of displaying in its original format, but i can just partition it in multiple lines.

Reply:
this syntax is updated.